### DISCUSSION TITLE
https://owner-api.teslamotors.com/api/1/vehicles return up to 11 vehicles

### DISCUSSION THREAD

**Discussion by wguan999** [timdorr/tesla-api#478] [Q&A] [answered]
HI, experts

I used POSTMAN to query https://owner-api.teslamotors.com/api/1/vehicles. I have 30+ vehicles under my accounts. This API call only returns 11 vehicles. Do you know how to resolve this issue?

Thanks.

**Comment by GaPhi** [comment]
Without having kind of account myself, it is difficult to help you but you can try to add an 'offset' field or, better, use the tesla application and check its logs (all queries are logged with their parameters).

When you have the answer, let share the information!

**Comment by wguan999** [comment, marked as answer]
I think the API returns all the vehicles that have been delivered (11 in my case). The rest are ordered but not delivered yet. These undelivered vehicles are shown in Tesla Web UI, but not included in the API calls.

**Reply by davidhodge** [reply]
That is standard / expected behavior. So that sounds right.
